src/olympia/activity/utils.py

```diff
@@ -152,21 +152,36 @@ def action_from_user(user, version):
         return amo.LOG.REVIEWER_REPLY_VERSION
 
 
-def log_and_notify(action, comments, note_creator, version):
+def template_from_user(user, version):
+    review_perm = ('Review' if version.channel == amo.RELEASE_CHANNEL_LISTED
+                   else 'ReviewUnlisted')
+    template = 'activity/emails/developer.txt'
+    if (not version.addon.authors.filter(pk=user.pk).exists() and
+            acl.action_allowed_user(user, 'Addons', review_perm)):
```

Comment:
This needs updating because of https://github.com/mozilla/addons-server/commit/4f9fa72213ed4f379230f01ea9672f5903271fa3, right ?